src/main/external-resources/renderers/Panasonic-VieraAS650.conf

```diff
@@ -17,7 +17,7 @@ RendererIcon = Panasonic-VieraAS650.png
 
 UserAgentAdditionalHeader = X-PANASONIC-Registration
 UserAgentAdditionalHeaderSearch = REFOVFYAAAAAAAAAAAAAAAA
-UpnpDetailsSearch = \d{2}AS650
+UpnpDetailsSearch = \\d{2}AS650
```

Comment:
Because of Jave rules, as explained below: (we've already tested it in an other commit before)
https://docs.oracle.com/javase/tutorial/java/data/characters.html
Or [here](https://books.google.fr/books?id=fu5HtixRje8C&pg=PA327&lpg=PA327&dq=java+%5C%5C+double+each+backslash+character&source=bl&ots=nKMB9x2mvA&sig=aWPE004azB5ZDMnWEo2Oh0Kkeo4&hl=en&sa=X&ved=0ahUKEwi9xrm2qqTVAhVnIcAKHVbRAqcQ6AEIQjAF#v=onepage&q=java%20%5C%5C%20double%20each%20backslash%20character&f=false) and [there](http://www.javaworld.com/article/3188545/learn-java/java-101-regular-expressions-in-java-part-1.html).